File: src/main/java/client/ClientManager.java
/*
 * Copyright (c) Grigory "vcusnx" Markovich, 2023.
 *
 * The Messenger Project.
 */

package client;


public interface ClientManager {

    /**
     * Connect to server
     */
    void connect();

    /**
     * Disconnect from server
     */
    void disconnect();
}

File: app/src/main/kotlin/client/ClientManager.kt
/*
 * Copyright (c) Grigory "vcusnx" Markovich, ISLANDS TEAM, 2023.
 *
 *
 */

package client

interface ClientManager {

    /**
     * Connect to the server
     */
    fun connect(address: String)

    /**
     * Disconnect from server
     */
    fun disconnect()
}